registry/registry.go

```diff
@@ -369,10 +369,6 @@ func keySignChallengeCommit(ctx *gin.Context, data *registrationData) (bool, map
 			}
 		}
 
-		// Since CLI/auto-registered namespace did not have site name as part of their registration
-		// Use their IP for an educated guess
-		ns.AdminMetadata.SiteName = ctx.ClientIP()
```

Comment:
Is this just being removed altogether?

Reply:
Yes, I realized this is not necessary because we want the user to explicitly set `Sitename`, and we use that value for the director to show it on the director web UI, and defaulting to an IP address doesn't help much